src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,206 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.cardOuter {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  padding: 20px;
+  margin: 200px 30px;
+  justify-content: center;
+  background-color: $white-bg;
+  color: $base-color;
+  position: relative;
```

Comment:
what `position:relative `is doing here, in `flex` based layout using position doesn't make much sense.?